portfolio/src/main/webapp/comments.html

```diff
@@ -49,33 +50,61 @@ <h3>Submit a comment!</h3>
             required="required"></textarea>
 
           <input name="submit" 
+            class="submit"
             type="submit" 
             value="Submit comment" />
         </form>
       </div>
-      <form onsubmit="fetchAndInsertDataIntoDOM()">
-        <label for="number-comments">
-          Enter the number of comments to display:
-        </label>
-        <input type="number"
-          id="number-comments"
-          name="number-comments"
-          value=""
-          min="0" />
-        <input name="submit"
-          type="submit"
-          value="View comments" />
-      </form>
-      <button type="button"
+      <div id="comment-filter">
+        <p class="comment-header">~comments~</p>
+        <form id="comment-filter-form" onsubmit="fetchAndInsertDataIntoDOM()">
+          <label for="number-comments">
+            Comments shown:
+          </label>
+          <select id="number-comments"
+            name="number-comments"
+            value=""
+          >
```

Comment:
Nit: I personally find it more readable to put the end bracket right after the last attribute

Reply:
Got it